docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-aws/install-on-aws.rst

```diff
@@ -48,6 +48,8 @@ Installation
 
    pip install determined
 
+.. include:: ../../../_shared/pip-install-determined.txt
```

Comment:
This is probably a nit, and I trust whatever your preference is as being the best choice. Maybe the name `pip-install-determined-note` makes it slightly more obvious what will be included.

Reply:
yes good point